backend/pom.xml

```diff
@@ -35,11 +35,6 @@
             <groupId>org.springframework.security</groupId>
             <artifactId>spring-security-oauth2-resource-server</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.keycloak</groupId>
-            <artifactId>keycloak-admin-client</artifactId>
-            <version>9.0.2</version>
-        </dependency>
```

Comment:
We can just cast to LinkedHashMap instead of ClientRepresentation which removes the dependency on the Keycloak Admin Client and gets rid of all the null attributes that were being returned due to the casting.

Reply:
I'm fine with this, but notably that's a Jackson configuration, whether or not null attributes are serialized. We might want it back one day if we decide to manipulate the response.